ush/ufsda/misc_utils.py

```diff
@@ -56,20 +56,23 @@ def create_batch_job(job_config, working_dir, executable, yaml_path):
     with open(batch_script, 'w') as f:
         f.write('#!/bin/bash\n')
         if scheduler[job_config['machine']] == 'slurm':
+            if job_config['machine'] == 'hera':
+                taskspernode = job_config.get("ntasks-per-node", 18)
```

Comment:
Does this PR need to add `ntasks-per-node` to the yaml files in `ush/examples/run_jedi_exe/`?

Reply:
No, the second argument is the 'default' option if it is missing in the YAML.

Reply:
But I suppose for an example it might be a good idea to include optional entries?

Reply:
I guess the default will suffice for most cases.  My concern is that some users will not know they can set `ntasks-per-node` because they only interact with the configuration yamls in `ush/examples/run_jedi_exe/`.   I agree that we don't want to clutter the configuration yamls with too much stuff.   Is there another way to inform users that they can adjust the number of tasks per node?

Reply:
Works for me!